### DISCUSSION TITLE
I cant sync between my databases

### DISCUSSION THREAD

**Discussion by elmoumenyassine** [Budibase/budibase#9465] [Help]
Hello All,

I have Indexed 3 MYSQL databases (A,B,C) in my budibase data interface. All of those 3 DB, there is a table USERS.

all i want to do is : 

- When a new user is registered in database A / Table USERS => Insert the new row in [ database B / Table USERS ] and in   [ database C / Table USERS ]

How can i do that please ?

I am not a MYSQL expert. So what can i do ? 

In the automation part : After adding the 3 MYSQL databases, it only shows the first added databases !!!! I dont understand.

All i found is this article : https://budibase.com/blog/data/how-to-integrate-multiple-databases/ 

Thank you in advance 

____________________
In need, all the USERS tables are like this : 

`id,name,email,area_code,mobile_number,email_verified_at,password,role,phone_number,address,lat,long,image,avatar,forgot_token,provider_id,remember_token,created_at,updated_at,is_affiliator,balance,deleted_at`

_____________________
@melohagan @andz-bb @Den4ik1992 @stearm
@aptkingston @shogunpurple @budibot
@Mitch-Budibase @mike12345567 @kzkondo

**Comment by melohagan** [comment]
Hey @elmoumenyassine 

Automations are the correct way to go for that use-case.

> After adding the 3 MYSQL databases, it only shows the first added databases !!!! 

Could you provide some screenshots to demonstrate what you're seeing please?
